client/js/components/procedure/basicSettings/DpEmailList.vue

```diff
@@ -106,17 +108,18 @@ export default {
   },
 
   methods: {
-    delete (index) {
-      this.emails.splice(index, 1)
-      this.updateExtraEmailAddress(index)
-    },
-
     addElement () {
       this.emails.push({
         mail: this.formFields.mail
       })
     },
 
+    deleteEntry (index) {
```

Comment:
nitpick: In order to have one function do one thing, I'd rename this to `handleDelete`, move `this.emails.splice(index, 1)` (back) to a `delete` function and call it here. I'm not sure if `this.updateExtraEmailAddress(index)` is still needed?

Reply:
like so?
e13ad3e588c43000eb5bbcbd90664bc9919e7c04